interfaces/builtin/pulseaudio.go

```diff
@@ -23,27 +23,157 @@ import (
 	"github.com/snapcore/snapd/interfaces"
 )
 
-const pulseaudioConnectedPlugAppArmor = `
-/etc/pulse/ r,
-/etc/pulse/* r,
+var pulseaudioConnectedPlugAppArmor = []byte(`
+capability dac_override,
+
+/etc/machine-id r,
+
 /{run,dev}/shm/pulse-shm-* rwk,
-owner /{,var/}run/user/*/pulse/ r,
+
+# FIXME Desktop-only
+owner /{,var/}run/user/*/pulse/ rwk,
 owner /{,var/}run/user/*/pulse/native rwk,
```

Comment:
@jhodapp Can you split those two lines out and combine with the snippet depending on if release.OnClassic is set?